src/rgw/rgw_pubsub.h

```diff
@@ -401,7 +402,7 @@ struct rgw_pubsub_topic {
   std::string policy_text;
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(4, 1, bl);
+    ENCODE_START(5, 1, bl);
```

Comment:
why did we change the version if the struct did not change?

Reply:
> why did we change the version if the struct did not change?

I think I did not revert this code after removing the bucket-notif-is struct. Will do it

Reply:
> why did we change the version if the struct did not change?

done